strawberryfields/compilers/gaussian_unitary.py

```diff
@@ -175,6 +176,17 @@ def compile(self, seq, registers):
                         [dict_indices[modes[0]], dict_indices[modes[1]]],
                         nmodes,
                     )
+                elif name == "sMZgate":
+                    exp_sigma = np.exp(1j * (params[0] + params[1]) / 2)
+                    delta = (params[0] - params[1]) / 2
+                    U = exp_sigma * np.array(
+                        [[np.sin(delta), np.cos(delta)], [np.cos(delta), -np.sin(delta)]]
+                    )
+                    S = expand(
+                        interferometer(U),
+                        [dict_indices[modes[0]], dict_indices[modes[1]]],
+                        nmodes,
+                    )
```

Comment:
This is great! Soo happy you did this.

Reply:
does this cover your previous suggestion, or is there another file where we can add this kind of thing?

Reply:
Nop, I deleted the previous suggestion was I saw this. Very thorough!